src/test/resources/features/filters.feature

```diff
@@ -0,0 +1,21 @@
+Feature: Filters feature
+
+  Background:
+    Given ADMINISTRATOR login to the Report Portal
+    When User navigates to FILTERS menu
+    Then User creates a new filter
+
+  @smoke @filter_is_great
+  Scenario: Verify Filters page contains some filters
+    Given User navigates to FILTERS menu
```

Comment:
there is the same step in background section, I believe you can remove this one

Reply:
Yes, thanks, removed

Reply:
Yes, but during creation a new filter user is redirected to Launcher page an stays on it

Reply:
got it